src/documents/models.py

```diff
@@ -1367,6 +1375,79 @@ class WorkflowActionType(models.IntegerChoices):
         verbose_name=_("remove all custom fields"),
     )
 
+    email_subject = models.CharField(
```

Comment:
I wonder about grouping email data and webhook data into 2 different models, with a foreign key back here.  It kind of feels cleaner to group those, and access just the data needed for the type?

Essentially some database normalization

Reply:
I'd consider doing it for the others now, as this model starts growing, but that would be annoying probably

Reply:
This one was definitely a _doozy_ as they say, but I think I got it. Yes it bothers me a lot the others aren't separated but maybe another PR (also worse cause they need to be migrated).

Let me know if this works.